core/Network/TLS/Handshake/Random.hs

```diff
@@ -24,13 +24,14 @@ serverRandom ctx chosenVer suppVers
       TLS12  -> ServerRandom <$> genServRand suffix12
       _      -> ServerRandom <$> genServRand suffix11
   | TLS12 `elem` suppVers = case chosenVer of
+      TLS13  -> ServerRandom <$> getStateRNG ctx 32
```

Comment:
Would you add a comment about this intension?

Reply:
I added comments about the whole downgrade conditions.